yghannam-workunit-fixes-for-aarch64

yghannam opened the pull request:
Simple fixes to workunit scripts for AArch64.

dachary commented:
@yghannam would you mind spliting these in three diferent pull requests ? 7b4dc13 can obviously be merged.  	ed3efb0 needs a bit more explaining. And 353199d + ed4116d require careful checking to verify importing config.guess this way is not going to disrupt anything.

yghannam commented:
@dachary 
I've split them up and I've added a short explanation to https://github.com/ceph/ceph/commit/ed3efb0b7e42e7bff251a20107719376ec8a6c34.

I'll close this request. Thanks.
